### ISSUE TITLE
Remove TravisCI

### ISSUE THREAD

**Issue by travissluka** [JCSDA-internal/soca#1029]
I think we're ready to retire TravisCI. As far as I know, it was only being used for soca, and soca-science.

- [x] remove the TravisCI hooks from the JCSDA github: @eap @climbfuji  (I don't have the permissions any more to do this)
- [x] issue PR to remove the code from the soca repo that was being used for travisCI  (@travissluka )

**Comment by climbfuji** [comment]
@travissluka Not sure if I get to this before my vacation next week, if not I can do it in the first week of April. Or @eap if time permits, but I understand this is not urgent.

**Comment by travissluka** [comment]
> @travissluka Not sure if I get to this before my vacation next week, if not I can do it in the first week of April. Or @eap if time permits, but I understand this is not urgent.

correct, not urgent. I just wanted to make sure this is documented while I'm in a "clean up soca" mood

**Comment by climbfuji** [comment]
@ytremolet @eap @gibbsp (Kammie doesn't have a github account registered in JCSDA-internal it seems, will ping her on slack)

Update 2024/04/26:
1. I canceled the travis CI subscription, it will expire May 11. See screenshot below.

2. I uninstalled the Travis CI app from the JCSDA-internal organization (it had access to only two repos, soca and soca-science)

<img width="1502" alt="image" src="https://github.com/JCSDA-internal/soca/assets/8006981/35006779-aed6-4b39-9000-837e51099c3d">
